.rubocop.yml

```diff
@@ -22,7 +22,7 @@ Style/FrozenStringLiteralComment:
 
 
 Metrics/BlockLength:
-  Max: 20
+  Max: 50
```

Comment:
why?
you can disable rubocop in one file if you need 
but this number is not ok here